### DISCUSSION TITLE
Beginner NodeRed: Integration HA sensors?

### DISCUSSION THREAD

**Discussion by LaMiyu** [edterbak/HeishaMoNR#62] [General]
Hello,

First of all: Thank you very much, your solution is really great.

I use HA and heishamon is already integrated here. I have now also set up node red, configured MQTT, I can see all the information in the dashboard. It works :)

unfortunately I am an absolute beginner in Nodered:
How do I get my HA temperature sensors integrated into Nodered? I tried it with the HA node "Current State", but in the dashboard I only see the temperature values ​​from the heat pump itself.

In addition, I have integrated the nodered dashboard directly as a website in the HA dashboard - unfortunately I need a user and password every time I call it up - can this be switched off?

One last thing:
when deploying I get an error message:
![image](https://user-images.githubusercontent.com/73053267/210072540-e9eb130c-dac4-452c-8501-b438cfbfad71.png)
The "[WP Control] T_Outside Input" function causes problems
![image](https://user-images.githubusercontent.com/73053267/210072737-f6acc035-d266-4ad7-a358-82dfe48e8411.png)

Many Thanks

**Comment by LaMiyu** [comment]
ok.. reading all FAQ would have been good :X

![image](https://user-images.githubusercontent.com/73053267/210075365-a45b405a-cbc1-41e8-a706-2768fdbbc5db.png)
However, I only get:
- Event Data
or
- Entity State

not the event state as described

**Comment by LaMiyu** [comment]
Hello,

I used the current beta version - Flow version 21.33 beta (experimental)
Here the error message WP Control T-Outside is gone.

And I used Entity State, so it works.

The version has been stable for me so far. (Use only WAR )

**Comment by edterbak** [comment]
Hi LaMiyu, 

Sorry for the lack of reply.
If there are issues, could you then open an issue. :) This is more in my view. I do not get notification on discussions here. 

But has everything worked out ?

**Reply by LaMiyu** [reply]
yes thanks :)
